src/main/java/Duke.java

```diff
@@ -1,10 +1,47 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String command;
+        String[] tasks = new String[100];
+        int taskNumber = 0;
+        boolean[] isDone = new boolean[100];
+
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+        loop: do {
+            Scanner input = new Scanner(System.in);
+            command = input.next();
+
+            switch(command) {
+            case "bye":
+                System.out.println("Bye. Hope to see you again soon!");
+                break loop;
+            case "list":
+                for (int i = 0; i < taskNumber; ++i) {
+                    String doneStatus;
+                    if (isDone[i]) {
+                        doneStatus = "[Done] ";
+                    } else {
+                        doneStatus = "[Not Done] ";
+                    }
+                    System.out.println(i+1 + "." + doneStatus + tasks[i]);
+                }
+                break;
+            case "done":
+                command = input.next();
+                int i = Integer.parseInt(command);
+                isDone[i-1] = true;
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("[Done] " + tasks[i-1]);
+                break;
+            default:
```

Comment:
Another possible suggestion is to add comment lines so that it's easier to follow through a block of code 😃